coursemapper-kg/recommendation/app/services/course_materials/recommendation/resource_recommender.py

```
@@ -88,257 +125,139 @@
-    data["nodes"] = ser_resources
-    data["edges"] = ser_realations
-
-    return data
-
-
-def save_data_to_file(data, file_path):
-    data.to_csv(file_path, sep="\t", encoding="utf-8")
+        return result
+
+
+    def _get_resources_by_main_concepts(self, mid: str): #, slide_id: str):
```

Ruff (0.12.2)

Missing return type annotation for private function `_get_resources_by_main_concepts`

(ANN202)